### PR TITLE
chore: prepare messages

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
## Description

This PR improves prepare command success messages with hinting user which command he should use next.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [ ] I have read the **CONTRIBUTION_GUIDE** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by kysely — COMMENTED** [review]
I'm not sure the instructions are helpful or relevant here. To me, the convention of using  `prepare:` prefix to each command tells me there will be more here.

I feel like the current authors know the general _process_ of authoring and the messages are adding a lot of noise.

- `prepare:profile`: it might take hours or days before you start writing the map
- `prepare:map`: the testing here is questionably the most useful. Hint to prepare:provider might be useful when called without an existing provider json
- `prepare:provider`: if someone is preparing Provider JSON, I think they know what they have to do next 🤷 

---

Then I don't understand the redundancy of user's input

> Created profile: "${profile}" at path: "${path}".

I think the following would be sufficient since you just ran the command where you chose to create _profile_ with the specific name

> Created ${path}.supr

Also 

> Now you can edit created file

made me laugh, honestly 😄 I don't think we need to be _that_ specific 😉

**Comment by kysely** [comment]
Or, if you'd like to provide some idea of the overall authoring process, let's discuss output like this
```
•----------•-----------◦------◦
Profile    Provider    Map    Test
```

... or in text version?

```
ADDED grid/ai/generate-text/profile.supr
TODO Provider.json
TODO map.suma
TODO map.test.ts
```

**Comment by Jakub-Vacek** [comment]
Yeah, current state of PR is kinda funny/od. 
My goal was to lead user to follow recommended order (profile -> provider -> map) of commands.
I will definitely remove Information about what was created (location will stay).

I like text overview of authoring process 👍

**Comment by martinalbert** [comment]
For me it would make sense to get simple instructions if needed and get suggestions on what to use next, so rather than explaining it in sentence keep it simple and formatted like this for example:
```
$ sf prepare:profile scope/profile
🆗 Created profile: "scope/profile" at path: "scope/profile.supr".

next command suggestions:
sf prepare:provider <provider-name>
sf prepare:map scope/profile <provider-name>
```

```
$ sf prepare:provider provider-name
🆗 Created provider: "provider-name" at path: "provider-name.json".
:warning: Edit .env file for your credentials

next command suggestions:
sf prepare:map <profileId> provider-name
sf prepare:test <profileId> provider-name
```

```
$ sf prepare:map scope/profile provider-name
🆗 Created map ...

next command suggestions:
sf prepare:test scope/profile provider-name
```

```
$ sf prepare:test scope/profile provider-name
🆗 Created test ...

Run created test with live traffic:
yarn test:record scope/profile/maps/provider-name

or with recorded traffic:
yarn test scope/profile/maps/provider-name
```

The instructions might not bring that much value to me since I know that I should add credentials or I know how to run a test in station but it might bring value to someone new to this cli. 

I’m also not sure about suggestions as they’re all not that long commands. In some cases where there is a long profile name, I would find them helpful (just copy instead of typing profile name again) but on the other hand here might be better some autocomplete instead of command suggestions. 🤷🏻‍♂️

File: src/common/messages.ts [hunk 1]
```diff
@@ -239,13 +239,48 @@ const init = {
...
+    provider: string,
+    path: string,
+    station?: boolean
+  ) => {
+    let testPath = `${profile}.${provider}.test.ts`;
+    if (station === true) {
+      testPath = `grid/${profile}/maps/${provider}.test.ts`;
+    }
+
+    return `Created test at path: "${path}"\n\nRun created test with live traffic:\nyarn test:record ${testPath}\nor with recorded traffic:\nyarn test ${testPath}`;
```
[the remark this answers is not in the document]
Comment (Jakub-Vacek) [reply src/common/messages.ts:269]: @kysely Yes it is. I can print these only when `--station` flag is used. And have something like:

`Created test at path: "${path}"\n\nFollow https://github.com/superfaceai/testing/blob/dev/README.md to run created test`;

[the remark this answers is not in the document]
Comment (kysely) [reply src/common/messages.ts:269]: IMO this makes more sense

**Review by kysely — APPROVED** [review]

**Pull request merged**